tests/rados/test_bluestore_features.py

```diff
@@ -96,6 +98,39 @@ def run(ceph_cluster, **kw):
 
     rados_object.change_heap_profiler_state(osd_list, "stop")
 
+    execution_time = config["execution-time"]
+    log.info(f"Checking osd memery growth for {execution_time} minutes")
+    # Setting the scheduled scrub parameters to enable scrubbing
+    if scrub_object.get_osd_configuration("osd_scrub_begin_hour") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_begin_hour", 0)
+        log.info("The osd_scrub_begin_hour is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_end_hour") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_end_hour", 0)
+        log.info("The osd_scrub_end_hour is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_begin_week_day") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_begin_week_day", 0)
+        log.info("The osd_scrub_begin_week_day is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_end_week_day") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_end_week_day", 0)
+        log.info("The osd_scrub_end_week_day is set to 0")
+
+    time_execution = datetime.datetime.now() + datetime.timedelta(
+        minutes=int(execution_time)
+    )
+    while datetime.datetime.now() < time_execution:
```

Comment:
We can collect once at beginning and once at the end.

Reply:
Logic implemented.